### DISCUSSION TITLE
Amazon ads api Purchased product reports data are different from https://advertising.amazon.de/reports same type

### DISCUSSION THREAD

**Discussion by JHanAlex** [amzn/ads-advanced-tools-docs#173] [Q&A] [answered]
I tried to compare data by api report and report made via advertising center ui. 
The report data for the same date are different from several rows.
I asc api report:
`<?php
$client = new Client();
$headers = [
  'Content-Type' => 'application/vnd.createasyncreportrequest.v3+json',
  'Amazon-Advertising-API-ClientId' => 'amzn1.application-oa2-client........',
  'Amazon-Advertising-API-Scope' => '3634181669979980',
  'Authorization' => 'Bearer Atza|IwEBICM........oX5-.......8J0j2ZTuRVRK1cuuMhKqvlERbDH-z-HH_iFXwpYb9c5yyd9lq359BY'
];
$body = '{
  "endDate": "2023-10-22",
  "configuration": {
    "adProduct": "SPONSORED_PRODUCTS",
    "columns": [
      "date",
      "portfolioId",
      "campaignId",
      "campaignName",
      "adGroupId",
      "adGroupName",
      "keywordId",
      "keyword",
      "keywordType",
      "advertisedAsin",
      "purchasedAsin",
      "advertisedSku",
      "campaignBudgetCurrencyCode",
      "matchType",
      "unitsSoldClicks1d",
      "unitsSoldClicks7d",
      "unitsSoldClicks14d",
      "unitsSoldClicks30d",
      "sales1d",
      "sales7d",
      "sales14d",
      "sales30d",
      "purchases1d",
      "purchases7d",
      "purchases14d",
      "purchases30d",
      "unitsSoldOtherSku1d",
      "unitsSoldOtherSku7d",
      "unitsSoldOtherSku14d",
      "unitsSoldOtherSku30d",
      "salesOtherSku1d",
      "salesOtherSku7d",
      "salesOtherSku14d",
      "salesOtherSku30d",
      "purchasesOtherSku1d",
      "purchasesOtherSku7d",
      "purchasesOtherSku14d",
      "purchasesOtherSku30d",
      "kindleEditionNormalizedPagesRead14d",
      "kindleEditionNormalizedPagesRoyalties14d"
    ],
    "reportTypeId": "spPurchasedProduct",
    "format": "GZIP_JSON",
    "groupBy": [
      "asin"
    ],
    "timeUnit": "DAILY"
  },
  "name": "DAILY_PURCHASED_PRODUCTS_GROUP_BY_ASIN",
  "startDate": "2023-10-22"
}';
$request = new Request('POST', 'https://advertising-api-eu.amazon.com/reporting/reports', $headers, $body);
$res = $client->sendAsync($request)->wait();
echo $res->getBody();
` 

and done the same report via ui the same dates the same country
I got 19 rows in ui report. And 14 via API.

Why?  Please check this as you can

**Comment by elizafarley** [comment, marked as answer]
Hi @JHanAlex, small variations in the data between the API and ads console are expected. You can read more [here](https://advertising.amazon.com/API/docs/en-us/guides/reporting/v2/advertising-console).

If you continue to notice large discrepancies, I recommend opening a [Jira ticket](https://amzn-clicks.atlassian.net/servicedesk/customer/portal/2/group/2) so the support team can investigate.

**Reply by JHanAlex** [reply]
Thanks for answer.
But in this page Report type comparison table shown for v2 reporting version. 
I use v3 version

How ui report associated with v3 api reports?

**Reply by elizafarley** [reply]
We have a backlog task to update this doc for version 3, but it is not yet available. I will update you here when it's ready. 

However, for this specific issue, I believe the issue is a technical bug, and I would suggest opening a Jira ticket.

**Reply by JHanAlex** [reply]
Thanks. Щиро дякую
